src/scripts.js

```diff
@@ -1,6 +1,35 @@
 // This is the JavaScript entry file - your code begins here
 // Do not delete or rename this file ********
 
+import User from '../src/User'
+import mock from '../src/data/mock' 
+// console.log("user data:", User)
+
+let user;
+
+// query selectors
+const userName = document.querySelector('.user-name')
+const userAddress = document.querySelector('.user-address')
+const userEmail = document.querySelector('.user-email')
+const userStride = document.querySelector('.user-stride')
+const userSteps = document.querySelector('.user-steps')
+
+window.addEventListener('load', displayRandomUser() )
+
+function getRandomIndex(array) {
+  return Math.floor(Math.random() * array.length);
+}
+
+function displayRandomUser() {
+  user = new User(mock.users[getRandomIndex(mock.users)])
+  userName.innerText = user.name
+  // userAddress.innerText = user.address
```

Comment:
Will need to add line 26/27 to the dom. We may have to update the dom in order to query select these.